frontend/src/bundles/overview/components/goal-widget/goal-widget.tsx

```diff
@@ -0,0 +1,54 @@
+import { Icon } from '~/bundles/common/components/components.js';
+import { type ValueOf } from '~/bundles/common/types/types.js';
+
+import { CircleProgress } from '../components.js';
+import { GoalTypes } from './enums/goal-types.enums.js';
+
+type WidgetProperties = {
+    goalType?: ValueOf<typeof GoalTypes>;
```

Comment:
first requered props

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/53876827/a8c82bc9-543a-4fec-9503-ea785f85e5c7)
This is after installing the font locally